mgmt/api/NetworkMessage.h

```diff
@@ -49,6 +49,8 @@ enum class OpType : MgmtMarshallInt {
   EVENT_UNREG_CALLBACK,
   EVENT_NOTIFY, /* only msg sent from TM to client */
   STATS_RESET_NODE,
+  HOST_STATUS_UP,
+  HOST_STATUS_DOWN,
```

Comment:
Put these at the end of the range so that you don't unnecessarily change all the other enum values.